app/assets/javascripts/discourse/app/lib/link-hashtags.js

```diff
@@ -0,0 +1,62 @@
+import { schedule } from "@ember/runloop";
+import { ajax } from "discourse/lib/ajax";
+import { replaceSpan } from "discourse/lib/category-hashtags";
+import { TAG_HASHTAG_POSTFIX } from "discourse/lib/tag-hashtags";
+
+const categoryHashtags = {};
+const tagHashtags = {};
+const checkedHashtags = [];
```

Comment:
Wouldn't it be faster if we used a hash `{}` instead of an array `[]`?

Reply:
Yes, that's right. I will use a `Set` instead.